### PR TITLE
[ODS-5980] Refactor Standard and Extension version optional parameters

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1 [hunk 1]
```diff
@@ -151,11 +151,11 @@ function Initialize-DevelopmentEnvironment {
 
         [Parameter(Mandatory=$false)]
         [ValidateSet('4.0.0', '5.0.0')]
-        [String] $StandardVersion = '5.0.0',
+        [String] $StandardVersion,
```
Comment (simpat-jesus) [inline Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1:154]: We will need to update https://techdocs.ed-fi.org/display/ODSAPIS3V71/Getting+Started+-+Source+Code+Installation to include standard and extension version on the initialization steps

**Review by simpat-jesus — CHANGES_REQUESTED** [review]
Replace hardcoded extension version by using this job https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/08315e57b7fde8849bf6406e17c1904a3889e471/.github/workflows/Pkg%20EdFi.Ods.Minimal.Template.TPDM.yml#L34

**Comment by semalaiappan** [comment]
@simpat-jesus  Review comments are fixed . Ready for review

**Review by simpat-jesus — APPROVED** [review]

File: .github/workflows/InitDev Postgres, MultiTenancy.yml [hunk 2]
```diff
@@ -33,42 +33,53 @@ env:
...
+       shell: pwsh
+       run: |
+         .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "../Ed-Fi-Extensions"
+     - name: Set StandardVersions
+       id: Set_StandardVersions
+       working-directory: ./Ed-Fi-ODS/
+       run: |
+         $output = .\build.githubactions.ps1 StandardVersions -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-ODS/Application/EdFi.Ods.Standard/EdFi.Ods.Standard.csproj"
+         echo "StandardVersions=$output" >> $env:GITHUB_OUTPUT
+         Write-host "StandardVersions is  $output"
```
Comment (stephenfuqua) [inline .github/workflows/InitDev Postgres, MultiTenancy.yml:63]: Same comment as on the other pull request. I know we've talked about it a month or two ago as well. Please update to use the PowerShell syntax on both lines.

Comment (stephenfuqua) [inline .github/workflows/InitDev Postgres, MultiTenancy.yml:63]: In fact, this `echo` command does not work as written in PowerShell Core. I just tested locally and this does not set a value to `$env:GITHUB_OUTPUT`.

```suggestion
          $env:GITHUB_OUTPUT = "StandardVersions=$output"
         Write-host "StandardVersions is  $output"
```

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**